quiz.js

```diff
@@ -26,6 +37,21 @@ if (square(2) !== 4 || cube(3) !== 27 || quad(4) !== 256) {
 
 /* your code starts here */
 
+function sum(numbers, twiceOrHalf) {
+  console.log(numbers);
```

Comment:
you had the right idea but we're trying to add the numbers together, not manipulate the amount of numbers that are in the array